### PR TITLE
feat: ADF-208 Assign permission for Item content creator

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Relates to https://oat-sa.atlassian.net/browse/ADF-208

### CODE REVIEW

File: migrations/Version202107270952231101_taoQtiItem.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                '--' . SetRolesAccess::OPTION_CONFIG,
+                self::CONFIG,
+            ]
+        );
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->throwIrreversibleMigrationException(
+            'This role should have been applied in the past, so we should not roll it back'
```
Comment (shpran) [inline migrations/Version202107270952231101_taoQtiItem.php:46]: What does this mean?

File: manifest.php [hunk 2]
```diff
@@ -88,6 +89,36 @@
         ['grant', 'http://www.tao.lu/Ontologies/TAOItem.rdf#AbstractItemAuthor', QtiCreator::class],
         ['grant', 'http://www.tao.lu/Ontologies/TAOItem.rdf#AbstractItemAuthor', QtiCssAuthoring::class],
         ['grant', TaoRoles::REST_PUBLISHER, ['ext' => 'taoQtiItem', 'mod' => 'RestQtiItem']],
+        [
+            'grant',
+            TaoItemsRoles::ITEM_CONTENT_CREATOR,
```
Comment (gabrielfs7) [inline manifest.php:94]: Lets not forget to change `composer.json`

**Review by gabrielfs7 — APPROVED** [review]

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
Do not forget to update `composer.json` with new versions before merging.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/tao/Main/login

**Comment by AntonNeviarouski** [comment]
QA re-verified on http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/tao/Main/login

**Pull request merged**